libs/datamodel/core/src/transform/ast_to_dml/db/walkers.rs

```diff
@@ -357,3 +389,118 @@ impl<'ast, 'db> PrimaryKeyWalker<'ast, 'db> {
         self.attribute.name
     }
 }
+
+#[derive(Copy, Clone)]
+pub(crate) struct ExplicitRelationWalker<'ast, 'db> {
+    pub(crate) field_a: RelationFieldWalker<'ast, 'db>,
+    pub(crate) field_b: RelationFieldWalker<'ast, 'db>,
```

Comment:
The nested walkers are ballooning the size of the struct a bit, does it buy us a lot of convenience?